examples/language-sdk-instrumentation/golang-push/rideshare/docker-compose.yml

```diff
@@ -8,27 +9,26 @@ services:
     - PARAMETERS_POOL_SIZE=1000
     - PARAMETERS_POOL_BUFFER_SIZE_KB=1000
     - OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
-    - OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=http://prometheus:9090/api/v1/otlp/v1/metrics
```

Comment:
You need to keep this environment variable for otlp metrics to work. Same for the other services.